tools/unix/generate_localizations.sh

```diff
@@ -68,8 +68,10 @@ cat "$STRINGS_PATH"/{strings,types_strings}.txt> "$MERGED_FILE"
 
 # Generate list of languages and add list in gradle.properties to be used in build.gradle in resConfig
 SUPPORTED_LOCALIZATIONS="supportedLocalizations="$(sed -nEe "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/\1/p" "data/strings/strings.txt" | sort -u | tr '\n' ',' | sed -e 's/-/_/g' -e 's/,$//')
-if [ "$SUPPORTED_LOCALIZATIONS" != "$(grep listLanguages "$OMIM_PATH/android/gradle.properties")" ]; then
-  sed -i 's/supportedLocalizations.*/'"${SUPPORTED_LOCALIZATIONS}"'/' "$OMIM_PATH/android/gradle.properties"
+if [ "$SUPPORTED_LOCALIZATIONS" != "$(grep supportedLocalizations "$OMIM_PATH/android/gradle.properties")" ]; then
+  GRADLE_PROPERTIES="$OMIM_PATH/android/gradle.properties"
```

Comment:
define GRADLE_PROPERTIES  before and use in if ?

Reply:
Good catch! PTAL